apps/dashboard_web/lib/dashboard_web/live/dashboard_live/layout.ex

```diff
@@ -51,8 +50,8 @@ defmodule DashboardWeb.DashboardLive.Layout do
   @impl true
   def handle_event(
         "remove-component",
-        %{"dc-id" => dc_id} = params,
-        %{assigns: %{dashboard: dashboard, dashboard_components: dashboard_components}} = socket
+        %{"dc-id" => dc_id} = _params,
```

Comment:
Ooh, this is a great way to silence warnings without losing a variable you may want later.